biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -198,9 +198,14 @@ public static ContentCapability getContentCapability(Resource resource) {
 
 	public static List<ContentCapability> getContentCapabilities(Resource resource) {
 		List<ContentCapability> result = new ArrayList<>();
-
-		for (Capability c : resource.getCapabilities(ContentNamespace.CONTENT_NAMESPACE)) {
-			result.add(as(c, ContentCapability.class));
+		if (resource == null) {
+			System.err.println("null resource");
```

Comment:
A print here is wrong. Log if you need to say something. And the next line dereferences resource so it would NPE anyway. Perhaps this print is part of your debugging?